packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -56,11 +58,11 @@ export interface WizardProps extends React.HTMLProps<HTMLDivElement> {
   /** Custom height of the wizard */
   height?: number | string;
   /** The wizard title */
-  title: string;
+  title?: string;
   /** The wizard description */
   description?: string;
   /** Callback function to close the wizard */
-  onClose?: () => void;
+  onClose: () => void;
```

Comment:
I changed onClose to be required because otherwise the wizard crashes.

We can either:
- keep it optional. In that case multiple null checks have to be added to the wizard.
- change it to required which would be a breaking change

Reply:
I removed the fix from this PR (as it is for a discussion) and created an issue instead: https://github.com/patternfly/patternfly-react/issues/2831